packages/jumpstarter/jumpstarter/common/utils.py

```
@@ -63,11 +66,7 @@ def launch_shell(
         unsafe: Whether to allow drivers outside of the allow list
     """
 
-    env = os.environ | {
-        JUMPSTARTER_HOST: host,
-        JMP_DRIVERS_ALLOW: "UNSAFE" if unsafe else ",".join(allow),
-        "PS1": f"{ANSI_GRAY}{PROMPT_CWD} {ANSI_YELLOW}⚡{ANSI_WHITE}{context} {ANSI_YELLOW}➤{ANSI_RESET} ",
-    }
+    env = os.environ | {JUMPSTARTER_HOST: host, JMP_DRIVERS_ALLOW: "UNSAFE" if unsafe else ",".join(allow)}
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)